go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/spf13/afero v1.14.0
 	github.com/spf13/cobra v1.9.1
 	github.com/spf13/pflag v1.0.7
+	github.com/stretchr/testify v1.10.0
```

Comment:
We are very careful with new deps.
We use ginkgo only for the tests, so I think we need to keep the project standard.
Do you mind revert it?